VEX/priv/guest_amd64_toIR.c

```diff
@@ -13490,26 +13490,33 @@ Long dis_ESC_0F__SSE2 ( Bool* decode_OK,
          delta += 1;
          /* Insert a memory fence.  It's sometimes important that these
             are carried through to the generated code. */
-         stmt( IRStmt_MBE(Imbe_Fence) );
+         stmt( IRStmt_MBE(Imbe_SFence) );
```

Comment:
What about adding an additional parameter to MBE statement - a fence kind? (Same as for Flush).